addons/website_event_crm_questions/tests/test_event_registration.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.event_crm.tests.common import TestEventCrmCommon
+
+
+class EventRegistrationCase(TestEventCrmCommon):
+
+    def test_event_registration_lead_description(self):
+        """ Ensure that the lead description is well formatted/escaped
+        when created from an event registration. """
+
+        questions = self.env['event.question'].create([{
+            'title': 'Text Input Question',
+            'question_type': 'text_box',
+        }, {
+            'title': 'Meal Type',
+            'question_type': 'simple_choice',
+            'answer_ids': [
+                (0, 0, {'name': 'Mixed'}),
+                (0, 0, {'name': 'Vegetarian'}),
+            ]
+        }])
+
+        self.event_0.write({
+            'question_ids': [(4, question.id) for question in questions]
+        })
+
+        customer_data = self.batch_customer_data[1]
+        customer_data['registration_answer_ids'] = [(0, 0, {
+            'question_id': questions[0].id,
+            'value_text_box': "<div>hello world</div>",
+        }), (0, 0, {
+            'question_id': questions[1].id,
+            'value_answer_id': questions[1].answer_ids[0].id,
+        })]
+
+        registration_values = dict(self.batch_customer_data[1], event_id=self.event_0.id)
+        registration = self.env['event.registration'].create(registration_values)
+        self.assertEqual(len(self.event_0.registration_ids), 1)
+        self.assertEqual(len(self.test_rule_attendee.lead_ids), 1)
+        lead = self.test_rule_attendee.lead_ids.filtered(lambda lead: registration in lead.registration_ids)
+
+        self.assertTrue('&lt;' not in lead.description, 'Description should not contain any escaped html.')
```

Comment:
Actually, why ? We don't expect people to enter html, so it should be escaped, or is it something else ?

Reply:
Sanitizing `value_text_box` misleads me, sorry :man_facepalming: 
I updated the test to check that only the answers are escaped and not the HTML around.